internal/cmd/server/flags_test.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Flags", func() {
+    ctx := context.TODO()
+    options := &slog.HandlerOptions{
+        Level: slog.LevelDebug,
+	}
+    handler := slog.NewJSONHandler(GinkgoWriter, options)
+    logger := slog.New(handler)
```

Comment:
This should probably go inside a `BeforeSuite` block. It won't make a difference in this case, but in general if you put it here it will be executed when Ginkgo is reading the test suite, even if it isn't going to execute it. On the other hand the `BeforeSuite` block is executed only when the tests are going to be executed.